SU-SWS/stanford_profile_helper
# D8CORE-3169: change styles for prepended mailto link

# Not READY FOR REVIEW

# Summary
Adjust styles for prepending mailto icon in WYSIWYG

# Needed By (Date)
Sprint close

# Urgency
not

# Steps to Test

1. Check out this branch
1. Check out https://github.com/SU-SWS/stanford_profile/pull/391
2. Import config
3. Rebuild cache (drush cr)
4. Create or visit a page with a mailto link in the WYSIWYG. 
5. Verify it looks and behaves as expected.

# Affected Projects or Products
D9Core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-3169
- Other PRs: https://github.com/SU-SWS/stanford_profile/pull/391
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
